Dockerfile

```diff
@@ -0,0 +1,8 @@
+# release version passed in on build, just default it temporarily
+ARG         RELEASE_VERSION=0.0.1
+
+FROM        node:13.10-alpine3.11
```

Comment:
i would prefer to use the latest LTS version, so `node:12.18-alpine3.11`

Reply:
Changed to LTS version